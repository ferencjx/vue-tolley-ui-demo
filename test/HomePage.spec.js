import { shallowMount } from '@vue/test-utils'

import HomePage from '../src/components/HomePage'

describe('<HomePage>', () => {
  it('renders the page', () => {
    const searchResultPageWrapper = shallowMount(HomePage)
    expect(searchResultPageWrapper).toMatchSnapshot()
  })
})
