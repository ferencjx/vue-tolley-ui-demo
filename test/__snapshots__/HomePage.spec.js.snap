// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage> renders the page 1`] = `
<div class="hello">
  <h1>Welcome to Demo Vue.js App</h1>
</div>
`;
